src/index.html

```diff
@@ -17,6 +17,83 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              class="menu__link is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              data-qa="hover"
+              class="menu__link"
+              href="#"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="#"
```

Comment:
The `href` attribute is currently set to `#`, which is a placeholder. Ensure that this is updated to the correct URL or path when the site is deployed.